machine.go

```diff
@@ -505,14 +505,19 @@ func (m *Machine) startVMM(ctx context.Context) error {
 
 		return err
 	}
+
+	// This goroutine is used to kill the process by context cancelletion,
+	// but doesn't tell anyone about that.
 	go func() {
-		select {
-		case <-ctx.Done():
-			m.fatalErr = ctx.Err()
-		case err := <-errCh:
-			m.fatalErr = err
-		}
+		<-ctx.Done()
+		m.stopVMM()
```

Comment:
Should we be ignoring this error?

Reply:
The error from stopVMM(). Yeah, I think we should not ignore the error. Let me fix.

Reply:
The last commit logs the error. Passing that to errCh seems harder than I thought, since the channel will be closed after the underlying process is finished.